cardano/tx.go

```diff
@@ -107,3 +108,45 @@ func isAddressInOutputs(outputs []cardanowallet.TxOutput, addr string) (int, uin
 
 	return -1, 0
 }
+
+func CreateOnlyFeeTx(
+	cardanoCliBinary string,
+	testNetMagic uint,
+	protocolParams []byte,
+	timeToLive uint64,
+	metadataBytes []byte,
+	feeTxInput *TxInputInfo,
+	output cardanowallet.TxOutput,
+) ([]byte, string, error) {
+	// ensure there is at least 1 fee input
+	if feeTxInput == nil || len(feeTxInput.Inputs) == 0 {
+		return nil, "", errors.New("no fee inputs provided for CreateOnlyFeeTx")
+	}
+
+	feeAmount := feeTxInput.Sum[cardanowallet.AdaTokenName]
+
+	builder, err := cardanowallet.NewTxBuilder(cardanoCliBinary)
+	if err != nil {
+		return nil, "", err
+	}
+
+	defer builder.Dispose()
+
+	builder.SetProtocolParameters(protocolParams).SetTimeToLive(timeToLive).
+		SetMetaData(metadataBytes).SetTestNetMagic(testNetMagic).AddOutputs(output)
+
+	builder.AddInputsWithScript(feeTxInput.PolicyScript, feeTxInput.Inputs...)
+
+	calcFee, err := builder.CalculateFee(feeTxInput.PolicyScript.GetCount() * 2)
```

Comment:
Why *2 ? Is this because we are always signing with mutlisig signers? If it is, please write a comment explaining